### DISCUSSION TITLE
ERROR: Could not find a version that satisfies the requirement requirement.txt (from versions: none)

### DISCUSSION THREAD

**Discussion by AureliusEdran97** [community/community#121806] [Programming Help]
### Select Topic Area

Product Feedback

### Body

Hallo i try to make run my  discord bot 24/7 i using python, i'm new in all of this and i have a problem every time the same.
ERROR: Could not find a version that satisfies the requirement requirement.txt (from versions: none)
ERROR: No matching distribution found for requirement.txt
 also ModuleNotFoundError: No module named 'discord'.
 i try al l
![1](https://github.com/community/community/assets/106976864/2d0c5993-99e5-41af-a0d0-822e9bfc7f81)
ot of solution and i have the file requirements,txt but i dont' know what to do for solve it.

**Comment by ebndev** [comment]
Hey @AureliusEdran97! Were you able to figure out this error? There have been quite a few folks posting about Discord bots lately so it would be cool to know more about your error / project if there's any updates :)

**Comment by greeeen-dev** [comment]
Hey! This happens when you don't use the `-r` flag, which tells pip you're installing modules listed in your `requirement.txt` file. Since you're using Pterodactyl panel, you probably put `requirement.txt` in the **Additional Python packages** field instead of the **Requirements file** field. Putting this value in the correct field in your server Startup settings would fix the problem.

**Comment by dannysigalovich** [comment]
Open the requirements.txt file and ensure it contains valid package you can try other version.
maybe try install the library in other version in the requirements like `discord==version_number`
`pip install discord.py
`or `pip install -r requirements.txt`
